app/views/layouts/application.html.erb

```diff
@@ -4,6 +4,12 @@
     <title>Human Kind</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
+  <link rel="stylesheet" href="https://unpkg.com/leaflet@1.5.1/dist/leaflet.css"
+   integrity="sha512-xwE/Az9zrjBIphAcBb3F6JVqxf46+CDLwfLMHloNu6KEQCAWi6HcDUbeOfBIptF7tcCzusKFjFw2yuvEpDL9wQ=="
+   crossorigin=""/>
+  <script src="https://unpkg.com/leaflet@1.5.1/dist/leaflet.js"
```

Comment:
Do you need this?  I'd rather see us manage all the dependencies through yarn and package.json

Reply:
It from the leaflet setup guide, removing it will mess up the leaflet map